.circleci/publish

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euv
+
+echo $GPG_KEY | base64 --decode | gpg --batch --import
+
+gpg --passphrase $GPG_PASSPHRASE --batch --yes -a -b LICENSE
+
+mill mill.scalalib.PublishModule/publishAll \
+    "$OSS_USERNAME":"$OSS_PASSWORD" \
+    "$GPG_PASSPHRASE" \
```

Comment:
As this is a public repository, what prevents people from submitting a PR containing code like `println(java.lang.System.getenv("GPG_PASSPHRASE"))` and looking into the logs to see the printed key?

Reply:
That's a very good question! Using environment variables is [one of two suggested solutions for storing secrets](https://support.circleci.com/hc/en-us/articles/360006717953-Storing-secret-files-certs-etc-). Forked pull request builds are currently disabled for scalanet and even when we enable them it is [possible to disable passing secrets to them](https://circleci.com/docs/2.0/oss/#pass-secrets-to-builds-from-forked-pull-requests) and that is the default option.

But if you prefer I can change the setup to use contexts, which allow restricting environment variables usage to particular users.

Reply:
> Forked pull request builds are currently disabled
That looks like the right choice from my side, thanks for the details.

Reply:
To be honest I haven't thought about that issue, I just used proposed solution. Only good CircleCI saved me from introducing a security bug.